zulip_bots/zulip_bots/lib.py

```diff
@@ -82,6 +84,23 @@ def contains(self, key):
         # type: (Text) -> bool
         return key in self.state_
 
+class NotADefaultCommand(Exception):
+    def __init__(self, command, supported):
+        # type: (Text, Sequence[Text])
+        self.msg = "'{}' is not a supported default command; options are: {}".format(
+            command, ", ".join([repr(s) for s in supported]))
+    def __str__(self):
+        return self.msg
+
+to_dispatch = {}
```

Comment:
This doesn't feel right, having `to_dispatch` being a module-level variable.

Reply:
Would you prefer this in eg. `ExternalBotHandler`? Or in a separate class?

Is this approach preferable to #66?